b2b-sample/guardio-insurance-sample-application/components/editUser/editUserComponent.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useState } from 'react';
+import { Field, Form } from 'react-final-form';
+import { Button, ButtonToolbar, Loader, Modal, useToaster } from 'rsuite';
+import FormSuite from 'rsuite/Form';
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from '../../util/util/frontendUtil/frontendUtil';
+import { errorTypeDialog, successTypeDialog } from '../util/dialog';
+
+import stylesSettings from '../../styles/Settings.module.css';
+import styles from '../../styles/util.module.css';
+import decodeEditUser from '../../util/apiDecode/settings/decodeEditUser';
+
+export default function EditUserComponent(props) {
+
+    const toaster = useToaster();
+
+    const [loadingDisplay, setLoadingDisplay] = useState(LOADING_DISPLAY_NONE);
+
+    const nameValidate = (name, errors) => {
+        if (!name) {
+            errors.name = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const emailValidate = (email, errors) => {
+        if (!email) {
+            errors.email = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const usernameValidate = (username, errors) => {
+        if (!username) {
+            errors.username = 'This field cannot be empty'
+        }
+        return errors;
+    }
+
+    const validate = values => {
+        const errors = {}
+        errors = nameValidate(values.name, errors);
+        errors = emailValidate(values.email, errors);
+        errors = usernameValidate(values.username, errors)
```

Comment:
Missing semicolon at the end of the line. Make the code consistent.

Reply:
Fixed after moving the project was moved to support typescript.

### Commit
- https://github.com/wso2/samples-is/commit/7a6e3479af10061b15a74bccfd0eb6585901ec89